src/material/tooltip/tooltip.ts

```diff
@@ -180,7 +180,15 @@ export class MatTooltip implements OnDestroy, OnInit {
       this.hide(0);
     } else {
       this._updateTooltipMessage();
-      this._ariaDescriber.describe(this._elementRef.nativeElement, this.message);
+      this._ngZone.runOutsideAngular(() => {
+        // The `AriaDescriber` has some functionality that avoids adding a description if it's the
+        // same as the `aria-label` of an element, however we can't know whether the tooltip trigger
+        // has a data-bound `aria-label` or when it'll be set for the first time. We can avoid the
+        // issue by deferring the description by a tick so Angular has time to set the `aria-label`.
+        Promise.resolve().then(() => {
```

Comment:
For the record, I'm not a fan of having to add this, but it's the least evil of all the alternatives (e.g. doing our own change detection in one of the `after*Checked` hooks to see if there's an `aria-label` or adding a `MutationObserver`).